diagnostic.md

```diff
@@ -32,8 +40,8 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- solution below -->
-
+x = 3;
+y = 10;
```

Comment:
The values of x and y are 3 and 10, respectively, after line 38, but the question asked for the values after _each_ line of code. Also, "Do the variables change?  Do their values?"